src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -82,12 +73,13 @@ class AccountController extends AbstractContactController
     protected $accountContactFieldDescriptors;
     protected $accountAddressesFieldDescriptors;
 
-    public function __construct()
+    public function setContainer(ContainerInterface $container = null)
```

Comment:
Why did you change this to `setContainer` instead of using the constructor? Adding these methods is not what this method is thought for.